intellij/ast/src/test/kotlin/motif/ast/intellij/IntelliJAnnotationTest.kt

```diff
@@ -33,6 +37,14 @@ class IntelliJAnnotationTest : LightCodeInsightFixtureTestCase() {
         psiElementFactory = PsiElementFactory.SERVICE.getInstance(project)
     }
 
+    override fun getProjectDescriptor(): LightProjectDescriptor {
+        return object : ProjectDescriptor(LanguageLevel.HIGHEST) {
```

Comment:
Can this be it's own class or a util since it's reused in a few spots?

Reply:
`ProjectDescriptor` is a protected inner class of `LightCodeInsightFixtureTestCase` so I can't factor out the whole class but I did move the jdk logic to a shared module.